CONTRIBUTING.adoc

```diff
@@ -7,28 +7,27 @@ what kind of changes are likely to be accepted; and what to expect from the Spri
 Please refer back to this document as a checklist before issuing any pull request; this will save time for everyone!
 
 == Code of Conduct
+
 This project adheres to the Contributor Covenant link:CODE_OF_CONDUCT.adoc[code of conduct].
-By participating, you  are expected to uphold this code. Please report unacceptable behavior to
-spring-code-of-conduct@pivotal.io.
+By participating, you  are expected to uphold this code. Please report unacceptable behavior to spring-code-of-conduct@pivotal.io.
 
 == Understand the basics
 
-Not sure what a *pull request* is, or how to submit one?  Take a look at GitHub's excellent documentation:
-https://help.github.com/articles/using-pull-requests/[Using Pull Requests] first.
+Not sure what a *pull request* is, or how to submit one?
+Take a look at GitHub's excellent documentation: https://help.github.com/articles/using-pull-requests/[Using Pull Requests] first.
 
-== Search JIRA first; create an issue if necessary
+== Search GitHub issues first; create one if necessary
 
-Is there already an issue that addresses your concern?  Search the
-https://jira.springsource.org/browse/INT[JIRA issue tracker] to see if you can find something similar.
-If not, please create a new issue before submitting a pull request unless the change is truly trivial, e.g. typo fixes,
-removing compiler warnings, etc.
+Is there already an issue that addresses your concern?
+Search the https://github.com/spring-projects/spring-integration/issues[GitHub issue tracker] to see if you can find something similar.
+If not, please create a new issue before submitting a pull request unless the change is truly trivial, e.g. typo fixes, removing compiler warnings, etc.
 
```

Comment:
Shouldn't we still ask them to search JIRA too (since we're not planning on migrating)?

Reply:
Yeah... Make sense. Will mention that, too